dsgrid/cli/registry.py

```diff
@@ -449,6 +449,16 @@ def register_project(
     "Use it when the mappings are already registered.",
     callback=_path_callback,
 )
+@click.option(
+    "-a",
+    "--autogen-reverse-supplemental-mappings",
```

Comment:
I don't think we've discussed a convention. The code makes more sense when the parameter is plural. `--autogen-reverse-supplemental-mappings` becomes `autogen_reverse_supplemental_mappings`. That's probably less important than the user interface. I think I could make an argument for either of these:
```
dsgrid registry submit-dataset \
    --autogen-reverse-supplemental-mappings geography \
    --autogen-reverse-supplemental-mappings subsector
```
```
dsgrid registry submit-dataset \
    --autogen-reverse-supplemental-mapping geography \
    --autogen-reverse-supplemental-mapping subsector
```
I tend to prefer the plural because it reminds the user that they can pass the option multiple times.

Reply:
I get it now--the option collects them all, so it's collecting mapping*s*. That's fine with me.